docs/self-hosted/oel/oauth2/migrate.mdx

```diff
@@ -0,0 +1,63 @@
+---
+id: migrate
+title: Migrate to Ory Enterprise License
+---
+
+# Upgrade Ory Hydra to Ory Hydra enterprise license (OEL)
+
+This guide outlines the steps to upgrade from Ory Hydra open source to Ory Hydra Enterprise License (OEL). Upgrading to OEL
+provides additional features and support for enterprise customers.
+
+## Prerequisites
+
+Before starting the upgrade process, ensure you meet the following requirements:
+
+1. **Valid Ory Enterprise License**: You must have purchased an Ory Enterprise License.
+2. **Access to the Ory Enterprise Docker Registry**: You need access to download the OEL Docker images.
+3. **Ory Hydra Version**: Your current Ory Hydra installation must be version 2.0 or higher.
+4. **Backup and Testing**: Create a backup of your current Ory Hydra database and test the migration on a test environment to
+   ensure compatibility and minimize risks.
+
+## Upgrade process
+
+Follow these steps to upgrade from Ory Hydra open source to Ory Hydra Enterprise License (OEL):
+
+### 1. Keep the open source version running
+
+Ensure that your current open source version of Ory Hydra is running.
+
+### 2. Run the OEL Image for database migration
+
+Use the Ory Hydra Enterprise License Docker image to run the `hydra migrate sql up` command against the database of the open
+source service. This step applies the necessary database migrations for OEL.
+
+```bash
+docker run --rm \
+  -e DSN=your_database_connection_string \
+  europe-docker.pkg.dev/ory-artifacts/ory-enterprise/hydra-oel:<version> \
+  migrate sql up -e -f config.yml
```

Comment:
Let's mention that the config here which is OS one is compatible with OEL and no changes are required - would be more clear for the end user, WDYT?

Reply:
👍